src/algo/depth-to-rgb-calibration/optimizer.cpp

```diff
@@ -1559,6 +1559,7 @@ void params::set_depth_resolution( size_t width, size_t height, rs2_ambient_ligh
     // Some parameters are resolution-dependent
     bool const XGA = (width == 1024 && height == 768);
     bool const VGA = (width == 640 && height == 480);
+
```

Comment:
Do you think it makes sense to add a check here that says "if not XGA and not VGA, throw an exception"?

Reply:
No its can theoretically support more resolutions , in this case it will get the default values.

Reply:
But we don't "support" other resolutions (like 1024x480) in our implementation...

Reply:
done